src/generated/typings/index.ts

```diff
@@ -30,5 +30,5 @@ export type SortOrder = "Asc" | "Dsc";
 /**
  * Track
  * represent a single track, id and path should be unique
- */
-export type Track = { _id: string, title: string, album: string, artists: Array<string>, genres: Array<string>, year: number | null, duration: number, track: NumberOf, disk: NumberOf, path: string, };
+*/
+export type Track = { _id: string, title: string, album: string, artists: Array<string>, genres: Array<string>, year: number, duration: number, track: NumberOf, disk: NumberOf, path: string, };
```

Comment:
Why remove the `null` from track? This typing is generated via `cargo test` and mimics the definition in Rust:

https://github.com/martpie/museeks/blob/master/src-tauri/src/libs/track.rs#L18-L30

so if you don't keep them in sync the CI will scream at you.

Reply:
changed back as it was before.